util/Order.cpp

```diff
@@ -277,7 +278,7 @@ FleetMoveOrder::FleetMoveOrder(int empire, int fleet_id, int start_system_id, in
         return;
     }
 
-    std::pair<std::list<int>, double> short_path = GetUniverse().ShortestPath(m_start_system, m_dest_system, empire);
+    std::pair<std::list<int>, double> short_path = GetUniverse().GetPathfinder()->ShortestPath(m_start_system, m_dest_system, empire);
```

Comment:
rather than two layers of getters everywhere, maybe have a free GetPathfinder() function available?